src/DotNetWorker/DotNetWorker.csproj

```diff
@@ -8,15 +8,16 @@
     <AssemblyName>Microsoft.Azure.Functions.Worker</AssemblyName>
     <RootNamespace>Microsoft.Azure.Functions.Worker</RootNamespace>
     <GenerateDocumentationFile>true</GenerateDocumentationFile>
-    <MinorProductVersion>8</MinorProductVersion>    
+    <MinorProductVersion>8</MinorProductVersion>
+    <PatchProductVersion>1</PatchProductVersion>
```

Comment:
I only bumped the patch version as the upgrade in dependencies is not bringing any public API changes/behavior changes to customers.  Open to discuss the SEMVERsioning of this change.

Reply:
Will update release notes after we collectively agree on the right package version bump to be used.